manifests/03-deployment.yaml

```diff
@@ -25,6 +25,11 @@ spec:
         control-plane: controller-manager
         controller-tools.k8s.io: "1.0"
     spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
```

Comment:
At least one of the CI failures is due to this line.

Reply:
@s-urbaniak this line is breaking the CI. Please take a look

Reply:
Indeed, I need to have a look once the re-revert of SCCs lands cc @stlaz